apps/Minecraft Pi (Modded)/install

```diff
@@ -1,37 +1,36 @@
 #!/bin/bash
-"${DIRECTORY}/api" apt_lock_wait
 
-# Remove old MCPILs
-sudo apt-get remove -y mcpil-r mcpil &>/dev/null || true
+# Version
+version="196/artifact/out/minecraft-pi-reborn-client-2.3.5"
 
-# Install the repos
-wget -qO- https://raw.githubusercontent.com/MCPI-Revival/mcpi-packages/master/install.sh | grep -v 'sudo apt update' | sudo bash - || error "Failed to install APT repo!"
+# Install
+case "${arch}" in
+    "64") do url_arch=arm64 ;;
+    "32") do url_arch=armhf ;;
+esac
+url="https://jenkins.thebrokenrail.com/job/minecraft-pi-reborn/job/master/${version}-${url_arch}.AppImage"
 
+# Cleanup Old Script Versions
+rm -f /etc/apt/trusted.gpg.d/mcpi-revival.gpg
+rm -f /etc/apt/sources.list.d/mcpi-revival.list
+sudo apt-get uninstall -y gmcpil jmcpil minecraft-pi-reborn-client || :
 
+# Download file
+mkdir -p ~/.local/bin || error "Could not make directory"
+wget -O ~/.local/bin/com.thebrokenrail.MCPIRebornClient.AppImage "${url}" || error 'Failed to download AppImage'
+chmod +x ~/.local/bin/com.thebrokenrail.MCPIRebornClient.AppImage
 
-# Choose MCPIL to use
-PS3="Which launcher for MCPI Modded would you like to use? Need help deciding? Ask here: https://discord.com/invite/aDqejQGMMy
-"
-options=("gMCPIL (recommended, unmaintained)" "jMCPIL (better GUI, unmaintained)" "Planet (Unofficial, but maintained)" "Quit")
-select opt in "${options[@]}"
-do
-  case $opt in
-    "${options[0]}")
-        MCPIL="gmcpil"
-        break;;
-    "${options[1]}")
-        MCPIL="jmcpil"
-        break;;
-    "${options[2]}")
-        MCPIL="planet-launcher"
-        curl -s --compressed "https://mcpiscript.github.io/PPA/KEY.gpg" | sudo apt-key add -
-        sudo curl -s --compressed -o /etc/apt/sources.list.d/mcpiscript.list "https://mcpiscript.github.io/PPA/mcpiscript.list"
-        sudo apt update
-        break;;
-    "${options[3]}")
-        exit 0;;
-    *) echo "Invalid option: $REPLY";;
-  esac
-done
-install_packages $MCPIL python3-pip python3-dev || error "Failed to install dependencies"
-pip3 install mcpi || error "The pip3 command failed to install mcpi!"
+enable_module fuse || exit 1
+
+# Make Desktop file
+mkdir -p ~/.local/share/applications || error "Could not make directory"
+echo "[Desktop Entry]
+Name=Minecraft: Pi Edition: Reborn (Client)
```

Comment:
This is an unusually long name and it will consequently widen the entire Games section in the start menu. Is the entirety of this name necessary?

Reply:
Changed